templates/project/.cargo/config.toml

```diff
@@ -3,30 +3,21 @@
 {% if mcu != "esp32" %}#{% endif %}target = "xtensa-esp32-espidf"
 {% if mcu != "esp32s2" %}#{% endif %}target = "xtensa-esp32s2-espidf"
 {% if mcu != "esp32s3" %}#{% endif %}target = "xtensa-esp32s3-espidf"
-{% if mcu != "esp32c3" %}#{% endif %}target = "riscv32imc-esp-espidf"
 
 [target.xtensa-esp32-espidf]
 linker = "ldproxy"
 runner = "espflash --monitor"
+rustflags = ["--cfg", "espidf_time32"]
```

Comment:
This espidf_time32 thing was needed on all platforms as long as we are on ESP-IDF 4, but in this file it was only being applied for the riscv target. It should be applied everywhere.